upgrades/1.3-1.4/common/AbstractMediaMigration.php

```diff
@@ -125,6 +127,7 @@ public function storeLocalMedias()
 
         $finder      = new Finder();
         $storedFiles = 1;
+        $batch = 500;
         foreach ($finder->files()->followLinks()->in($this->mediaDirectory) as $file) {
             $fileInfo = $storer->store($file, FileStorage::CATALOG_STORAGE_ALIAS);
             $em->clear();
```

Comment:
we already clear it for each file but without detaching, with the bulk clear, this one is useless imho, you can drop it. doing  this we'll clear each 500 files and not each file (should be faster but should consume more memory)